src/Polly.Caching/Polly.Caching.csproj

```
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net8.0;net6.0;netstandard2.0;net472;net462</TargetFrameworks>
+    <AssemblyTitle>Polly.Caching</AssemblyTitle>
+    <RootNamespace>Polly</RootNamespace>
+    <Nullable>enable</Nullable>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <ProjectType>Library</ProjectType>
+    <UsePublicApiAnalyzers>false</UsePublicApiAnalyzers>
+    <EnablePackageValidation>false</EnablePackageValidation>
+    <LegacySupport>true</LegacySupport>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Description>Polly.Caching provides caching strategies for Polly v8 built on Microsoft.Extensions.Caching.*.</Description>
```

Once the PR is further along, the "built on" will likely need updating and made more specific, as this implies it works with anything in that namespace.

```suggestion
    <Description>Polly.Caching provides caching strategies for Polly.Core built on Microsoft.Extensions.Caching.*.</Description>
```